website/docs/users-sources/sources/social-logins/google/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Google Identity Providers
+sidebar_label: Google
+tags: [integration, saml, google]
+---
+
+Google provides several services that can be used as identity providers in authentik.
```

Comment:
I think this is a good place to make it clear that these docs are about how to use the service as a Source... that these docs are about setting up authentik to use the credentials from the service. I guess that saying the "services can be used as identity providers is accurate, but we need to highlight that the main goal is to use the service's existing collection (source) of credentials to authenticate users.

Reply:
Also thanks for adding this top-level page!